USNRewardPayOut.sol

```diff
@@ -0,0 +1,32 @@
+import "./SampleOffer.sol";
+
+
+contract USNRewardPayOut {
+     
+     SampleOffer public usnContract;
+     
+     function USNRewardPayOut(SampleOffer _usnContract) {
+          usnContract = _usnContract;
+     }
+
+     // interface for USN
+    function payOneTimeReward() returns(bool) {
```

Comment:
It has to do with our proposal to the Universal Sharing Network. This contract ... the `USNRewardPayout` won't be deployed at the moment, it will just use the deployed offer so it may change later.

The difference is that the nodes of the USN , the EC(Ethereum Computers) can either pay an one time Fee to the DAO or just pay a percentage.

We will work on this further in the future, and keep it a separate contract to give us the flexibility to do so, plus to avoid the "DAO using the offer to generate endless reward tokens" problem.